### PR TITLE
remove axios vendoring

### PR DESCRIPTION
**Pull request by Khaledgarbaya**
Fxing https://github.com/contentful/contentful-sdk-core/issues/3

## TODO
- [x] Remove Nodejs 0.10 support
- [x] Pick solution 2 mentioned in the [README](https://github.com/contentful/contentful-sdk-core#future-incompatibility-between-lodash-and-core-js)

### CODE REVIEW

**Comment by axe312ger** [comment]
we continue with https://github.com/contentful/contentful-sdk-core/pull/10

**Comment by axe312ger** [comment]
okay lets avoid crazy merges and keep the chain simple -> reopening

**Comment by Khaledgarbaya** [comment]
removed all the spammy coveralls messages

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+module.exports = {
+    "extends": "standard",
+    "plugins": [
+        "standard",
+        "promise"
+    ]
+};
```
Comment (Khaledgarbaya) [inline .eslintrc.js]: standard 👉🏼  ~`;`~

File: webpack.config.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
+'use strict'
```
Comment (Khaledgarbaya) [inline webpack.config.js]: didn't we remove webpack, why this is here ?

**Comment by Khaledgarbaya** [comment]
ok Rebased master and resolved some conflicts with master

**Pull request merged**